README.md

```diff
@@ -1,7 +1,8 @@
-This maven plugin permits registration of a service with a consul agent.
+This maven plugin permits registration of a service with a consul agent. 
 
 Requirements: a consul agent must be running and listening on the service host.
 
 Invocation:
-
+		
```

Comment:
Why doesn't this tool add the capability to ignore whitespace at the click of a button?